--- /lxqt-config-appearance/main.cpp
+++ /lxqt-config-appearance/main.cpp
@@ -34,7 +34,6 @@
 #include "styleconfig.h"
 #include "fontsconfig.h"
 
-#include "../liblxqt-config-cursor/selectwnd.h"
 
 int main (int argc, char **argv)
 {
@@ -62,10 +61,6 @@
     dialog->addPage(fontsPage, QObject::tr("Font"), QStringList() << "preferences-desktop-font" << "preferences-desktop");
     QObject::connect(dialog, SIGNAL(reset()), fontsPage, SLOT(initControls()));
 
-    SelectWnd* cursorPage = new SelectWnd(sessionSettings, dialog);
-    cursorPage->setCurrent();
-    dialog->addPage(cursorPage, QObject::tr("Cursor"), QStringList() << "input-mouse" << "preferences-desktop");
-
     dialog->setAttribute(Qt::WA_DeleteOnClose);
     dialog->setWindowIcon(QIcon::fromTheme("preferences-desktop-theme"));
     dialog->show();
